public/styles/top_p.css

```diff
@@ -11,6 +11,19 @@ table {
 	border-right: 1px solid #98312A;
 }
 
-h2 {
-    margin-top: 0.5em;
```

Comment:
I added this to avoid the huge spacing of the title, see here "Description"

![image](https://cloud.githubusercontent.com/assets/4370550/13947413/18476d7e-f023-11e5-9c58-6f2e9c47d545.png)

Probably the correct way to go is to convert this into a `<h1>` and fix the styles there?